airbyte_rock/rockcraft.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: airbyte
+summary: Airbyte rock
+description: Airbyte OCI image for the Airbyte charm
+version: "1.0"
+base: ubuntu@22.04
+license: Apache-2.0
+platforms:
+  amd64:
+
+environment:
+  JAVA_HOME: /usr/lib/jvm/java-21-openjdk-amd64
+  CDK_PYTHON: /usr/bin/python3.10
+  CDK_ENTRYPOINT: /usr/lib/python3.10/dist-packages/airbyte_cdk/connector_builder/main.py
+  CDK_VERSION: "5.12.0"
+
+
+parts:
+  install-dependencies:
+    plugin: nil
+    stage-packages:
+      - apt-transport-https
+      - ca-certificates
+      - curl 
+      - gnupg
+      - python3.10-venv  
+    override-build: |
+      # Install kubectl
+      echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.31/deb/ /' | tee /etc/apt/sources.list.d/kubernetes.list
+      chmod 644 /etc/apt/sources.list.d/kubernetes.list   # helps tools such as command-not-found to work correctly
+      curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.31/deb/Release.key | gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+      chmod 644 /etc/apt/keyrings/kubernetes-apt-keyring.gpg # allow unprivileged APT programs to read this keyring
+      apt-get update
+      apt-get install -y kubectl
```

Comment:
Can this not be a build-package?

Reply:
It actually needs to be included in the final build as the application itself uses `kubectl`. Including it as a `stage-snap` did not work for some reason, so I opted to install it manually.